qaframework-bdd-tests/src/test/resources/features/refapp-2.x/07-visit/visitTypes.feature

```diff
@@ -31,3 +31,4 @@ Feature: Visit types Management
   Scenario: Deleting a Visit Type Test
      And User clicks on already saved visit type
      Then User clicks on delete visit type forever button
+
```

Comment:
Thank you @Muta-Jonathan for putting this together. Normally that last line should not be read as one that has code (space character). To created it, just press enter after the word button on line 33 and then backspace to the very start of the line. There it will not show as an added line with a +. It will be there but when you can not visibly see it like it is now. I don't know if this makes sense to you.

Reply:
oh